src/Utilities/FractionUtilities.hpp

```diff
@@ -157,7 +156,8 @@ Fraction simplest_fraction_in_interval(const T1& end1,
       if (++cf2) {
         ++term2;
       }
-      result.insert(gsl::narrow<Term_t>(std::min(term1, term2)));
+      using std::min;
```

Comment:
Why is this better than just calling `std::min`?

Reply:
To be honest, there's not much of a reason to do it for `std::min` because it has a nice general templated implementation.  The advantage in general is that if you write your own `min(SpecialClass a, SpecialClass b)` for some reason, this can call that instead of `std::min` if you follow [some rules about namespaces](https://en.cppreference.com/w/cpp/language/adl).  It's more important for things like `abs` where the standard doesn't provide a general version.

I'll drop this hunk if you prefer, since I can't actually imagine a case where it would be useful.